app/cruds/cruds_recommendation.py

```diff
@@ -0,0 +1,72 @@
+from typing import Sequence
+
+from sqlalchemy import delete, select, update
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models import models_recommendation
+from app.schemas import schemas_recommendation
+
+
+async def get_recommendation(
+    db: AsyncSession,
+) -> Sequence[models_recommendation.Recommendation]:
+    result = await db.execute(select(models_recommendation.Recommendation))
+    return result.scalars().all()
+
+
+async def create_recommendation(
+    recommendation: models_recommendation.Recommendation,
+    db: AsyncSession,
+):
+    db.add(recommendation)
+    await db.commit()
+    return recommendation
+
+
+async def update_recommendation(
+    recommendation_id: str,
+    recommendation: schemas_recommendation.RecommendationEdit,
+    db: AsyncSession,
+):
+    result = await db.execute(
+        update(models_recommendation.Recommendation)
+        .where(models_recommendation.Recommendation.id == recommendation_id)
+        .values(**recommendation.model_dump(exclude_none=True))
+    )
+    if result.rowcount == 1:
+        await db.commit()
+    else:
+        await db.rollback()
+        raise ValueError
+
+
+async def delete_recommendation(
+    recommendation_id: str,
+    db: AsyncSession,
+):
+    result = await db.execute(
+        delete(models_recommendation.Recommendation).where(
+            models_recommendation.Recommendation.id == recommendation_id
+        )
+    )
+    if result.rowcount == 1:
+        await db.commit()
+    else:
+        await db.rollback()
+        raise ValueError
+
+
+async def get_recommendation_by_id(
+    recommendation_id: str,
+    db: AsyncSession,
+) -> models_recommendation.Recommendation:
+    result = await db.execute(
+        select(models_recommendation.Recommendation).where(
+            models_recommendation.Recommendation.id == recommendation_id
+        )
+    )
+    try:
+        return result.scalars().one()
+    except NoResultFound:
+        raise ValueError
```

Comment:
Does the exception returned by the crud really matters ? I think what matters more is the HTTP exception sent by the endpoint.

Also I don't know which exception would be more appropriate than `ValueError` in a situation where the recommendation id doesn't exist.

Reply:
It does not really matter, but by convention in Hyperion, I would suggest to use `Recommendation | None`

Reply:
The problem is that the cruds update and delete also raise a `ValueError` when the recommendation does not exist and I cannot use `Recommendation | None` for these two cruds as they don't return values.
So I agree it would be better to follow the convention used in Hyperion but it wouldn't make much sense to have one crud which return `Recommendation | None` and the two others that return `ValueError`.

Reply:
I would say update and delete cruds should never be called when the object does not exist, while the get crud can

Reply:
I made the change. It will now return None if the object is not found.